src/passes/hash-stringify-walker.cpp

```diff
@@ -79,15 +78,22 @@ HashStringifyWalker::makeRelative(uint32_t idx) const {
   // The upper_bound function returns an iterator to the first value in the set
   // that is true for idx < value. We subtract one from this returned value to
   // tell us which function actually contains the the idx
-  uint32_t funcStartIdx = *--funcIndices.upper_bound(idx);
-  return {idx - funcStartIdx, idxToFuncName.at(funcStartIdx)};
+  auto itr = *--idxToFuncName.upper_bound(idx);
```

Comment:
```suggestion
  auto [funcIdx, func] = *--idxToFuncName.upper_bound(idx);
```

Could this be simplified if we use `lower_bound`?

Reply:
done